Test/Integration/Page/MultiStorePagesTest.php

```diff
@@ -80,7 +80,9 @@ public function testMultiStorePageIndices()
         $this->pageRepository->save($homePage);
 
         $this->pagesIndexer->execute([self::HOME_PAGE_ID]);
-        $this->algoliaHelper->waitLastTask();
+
+        $this->algoliaHelper->waitLastTask($defaultStore->getId());
```

Comment:
We use a lot of empty `waitLastTask()` calls. Should we start scoping those?

Reply:
I forgot to scope some of them in the multi-stores tests, so I'll do it, thanks !
I think it's also better to keep them unscoped in the other tests to make sure the original implementation still works for "simple application website".